TrajectoryPredictor.py

```diff
@@ -194,7 +194,7 @@ def draw_points_to_frame(self, points: np.array, frame: np.array,
         # Draw each point
         for pt in pts:
             x, y = pt[0]
-            if x is not None and x >= 0 and y >= 0 and x <= frame.shape[1] and y <= frame.shape[0]:
+            if x >= 0 and y >= 0 and x <= frame.shape[1] and y <= frame.shape[0]:
```

Comment:
how about... `0 <= x <= frame.shape[1] and 0 <= y <= frame.shape[0]`? (Test this tho...)